contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1880,6 +1962,9 @@ class tsqlMutator : public TSqlParserBaseListener
 public:
     MyInputStream &stream;
 
+	std::vector<int> double_quota_places;
+	int parameterIndex = 0;
```

Comment:
1. indentation & consistent naming convention (camelCase of camel_case)
2. What happens if we have more than 2 execute statements in a single batch? Could `executeParameterIndex` be a better name?

Reply:
if we have two 
for example like 
exec a 0,1,2,3
exec b 0,1,2,3
then it'll go from  0,1,2,3,4,5,6,7